### DISCUSSION TITLE
Implementation of CF Auth with HVAC

### DISCUSSION THREAD

**Discussion by ssrinivasreddys** [hvac/hvac#976] [General]
I am having Python apps running in PCF (Pivotal Cloud Foundry) which need to integrate with HashiCorp Vault for reading secrets. The authentication leverages 'cf auth' which is certificate based authentication. Please refer this official documentation from HashiCorp https://developer.hashicorp.com/vault/api-docs/auth/cf#login. Do we already have this supported in hvac? If yes, can you provide me leads please. 

Thanks

**Comment by briantist** [comment]
Hi @ssrinivasreddys it does not appear that we have any support for that in `hvac` so far. I've opened an issue for that in case you or someone else would like to implement it, thanks!
- #977

**Reply by ssrinivasreddys** [reply]
Hi @briantist Sure, thank you.
